domains/transfer/Transfer.API/API.IntegrationTests/Transfer/Api/Controllers/TransferAgreementsControllerTest.cs

```diff
@@ -580,6 +580,63 @@ await SeedTransferAgreements(
         updatedTransferAgreement!.EndDate.Should().Be(newEndDate);
     }
 
+    [Fact]
+    public async Task CreatePOATransferAgreement_ShouldCreateTransferAgreement_WhenInputIsValid()
+    {
+        var sub = Guid.NewGuid();
+        var orgId = Any.OrganizationId();
+        var senderName = "Some Awesome Org";
+
+        var request = new CreateTransferAgreementRequest(sub, orgId.Value, DateTimeOffset.UtcNow.ToUnixTimeSeconds(), DateTimeOffset.UtcNow.ToUnixTimeSeconds(), "12345678", "lol", "87456123", senderName);
```

Comment:
ReceiverOrganizationId = sub and Guid SenderOrganizationId = own org id???

Reply:
yes. Will rename to simplify reading of test :)